adminSiteClient/EditorCustomizeTab.tsx

```diff
@@ -376,6 +379,50 @@ class TimelineSection extends React.Component<{ editor: ChartEditor }> {
     }
 }
 
+@observer
+class MissingDataSection extends React.Component<{ editor: ChartEditor }> {
+    @computed get grapher() {
+        return this.props.editor.grapher
+    }
+
+    get missingDataStrategyOptions(): {
+        value: MissingDataStrategy
+        label: string
+    }[] {
+        const missingDataStrategyLabels = {
+            [MissingDataStrategy.auto]: "Automatic",
+            [MissingDataStrategy.hide]: "Hide charts with missing data",
```

Comment:
We have this slight problem that in our lingo, "chart" usually refers to the whole chart chrome (i.e. grapher as a whole), which is a bit confusing here.
I'm trying to come up with alternative words, but it's tough: plot, entities, visualisation, ...

Reply:
Ahh, okay. I think I'm leaning towards entities then? But it's a difficult one...

Reply:
I said "entities" for now but we can sleep on it :)